packages/fast-markdown-msft-react/src/rule.ts

```diff
@@ -0,0 +1,58 @@
+import {
+    IToken,
+    ITokens
+} from "./index";
+import { MarkdownIt } from "markdown-it";
+
+export interface IState {
+    tokens: IToken[];
+    Token: (type: string, tag?: string, idx?: number) => void;
+}
+
+export default class CreateRule {
+
+    constructor(md: MarkdownIt) {
+        return (this.replace as any);
+    }
+
+    public replace = (state: IState): void => {
+        this.renderComponents(state);
+    }
+
+    private renderComponents = (state: IState): void => {
```

Comment:
Can you leave a comment somewhere in this file explaining what it is doing?

Reply:
resolved